api/src/main/resources/workbench-api.yaml

```diff
@@ -2432,6 +2432,41 @@ paths:
           description: The Terra Job for the extraction workflow
           schema:
             "$ref": "#/definitions/GenomicExtractionJob"
+  "/v1/workspaces/{workspaceNamespace}/{workspaceId}/cohorts/abortExtract/{wgsCohortExtractionJobId}":
+    post:
+      parameters:
+        - in: path
+          name: workspaceNamespace
+          description: workspaceNamespace
+          required: true
+          type: string
+        - in: path
+          name: workspaceId
+          description: workspaceId
+          required: true
+          type: string
+        - in: path
+          name: wgsCohortExtractionJobId
```

Comment:
We moved away from this naming with my last PR. I'd just call this `jobId` now. Let me know if there were existing references that I missed